### ISSUE TITLE
Error in paper Equation

### ISSUE THREAD

**Issue by adam-ce** [hbb1/2d-gaussian-splatting#179]
I believe there is an error in Equation 9 of the paper. Imo it should say (u, v, 0, 1)^T there.

**Comment by hbb1** [comment]
The third entry is not used as the third column of **H** is zero (as we assume zero scaling in the third axis). We opt for (u,v,1,1) because it implies (uw,vw,w,1) where w = 1, similar to the representation of (xz,yz,z,1) for the image plane system.

**Comment by adam-ce** [comment]
I agree that it's not used in Equation 4 and 5 and I still believe that it is used to get from Eq. 9 to Equation 10.

**Comment by hbb1** [comment]
It does matter what is w is since it is a homogenous representation. I understand your confusion but I believe there is no such implication since for  the image plane we also use z=1 instead of z=0?

**Comment by adam-ce** [comment]
Trying to verify and understand Eq 10, I wrote down Eq 9 and solved for u and v. I arrived at something different than Eq 10 (there were some additional factors). So I investigated further. Changing Eq 9 to (u, v, 0, 1) maintains the homogeneous representation (last coordinate = 1) and results in your Eq 10. I also use Eq. 10 in my code and it seems correct.

To my understanding, this deals with the local splat coordinate system, not the image plane.

**Comment by hbb1** [comment]
Ohhhh, I see. Because hu^3 and hv^3 is already zero, which is very implicit that may lead to some confusions.  Therefore, using (u,v,1,1) and (u,v,0,1) are mathematically equivalent. The reason that using (u,v,1,1) instead of (u,v,0,1) is because the equation is from ray-quadratic intersection. And our equation is a degenerate case when the z scale degenerate to zero.

**Comment by adam-ce** [comment]
"Therefore, using (u,v,1,1) and (u,v,0,1) are mathematically equivalent." => they are in Eq 4/5 for the reasons you said, but 
imho not when going from Eq 9 to 10.

**Comment by adam-ce** [comment]
Ah, now I understand. yeah. I agree. Yeah, I'm using a different H matrix. Thanks for the clarification!
